nbtlib/tag.py

```diff
@@ -333,21 +337,44 @@ class ByteArray(Array):
     array_prefix = 'B'
     item_suffix = 'B'
 
+class abstring(type):

```

Comment:
still has a problem.
`nbtlib.List([nbtlib.String(b""),nbtlib.String("")])`
will make the second item `'""'`.
because String is not considered as a sub class of MalformedString.

should I change that into

```
def __instancecheck__(s,d):
		return super().__instancecheck__(d) or type.__instancecheck__(MalformedString,d)

```
?